js/main.js

```diff
@@ -1,2 +1,179 @@
 $(function () {
+
+// Rockpaperscissorsgame logic
+
+// users presses choice
+//bot generates number
+//compare and score game
+//increment player score
+//keep track of total
+//display results
+//reset display when new choice is made
+
+const choices = ["rock","paper",'scissors']  // bot choices
+
+let playerScore = 0
+let botScore = 0
+let playerBonus = 0
+let botBonus = 0
+let playerRoundScore = 0
+let botRoundScore = 0
+let playerGameScore = 0
+let botGameScore = 0
+let gameCounter = 10
+let output = ""  // status output
+let outputRound= "" // status of Round
+
+$('.scoreboardround').hide()
+$('.scoreboardgame').hide()
+$('#playagain').hide()
+$('#reloadgame').hide()
+
+
+$(".choice-buttons").on("click", function (e) {
```

Comment:
@thetripler currently this callback function is handling everything inside, which makes sense. consider breaking the code up into functional parts for brevity and to avoid repeating similar logic.
Great use of comments. helps with identifying the step by step procedures 👍